CONTRIBUTION.md

```diff
@@ -0,0 +1,39 @@
+Branching names:
+---
+Each branch must have a name in the following format with a proper prefix and number of Github's ticket it refers to:
+
+```
+prefix/000-branch-name
+``` 
+
+Allowed prefixes:
+* feature
+* bugfix
+* docs
+* test (when adding missing tests)
+* ...
+
+Commits:
+---
+Each commit (except for merge commits) must have a name in the following format:
+
+```
+000 - description of what was changed
+```
+
+Pull requests:
+---
+Each pull request must have a name in the following format where `000` represents a ticket number which this PR refers 
+to:
+```
+000 - description of what was changed
```

Comment:
I lately found myself in need to progressively lay out a task solution through multiple PRs, to avoid an overblown single PR.
What about using a common convention for those "progressive developments" like 
`000[-0] - brief and clear description of the actual change`

So that it would be easier to figure out the sequence of steps needed to merge the whole task solution?

Reply:
That would be fine, maybe start from 1 instead of 0 though.

Reply:
Actually that was a lapsus, I didn't meant to start from 0 (damn array programming 😁 !)